src/store/products/products.reducer.ts

```diff
@@ -0,0 +1,17 @@
+import { AppAction } from "../rootReducer.js";
+
+import { ProductsState, PRODUCTS_FETCH_SUCCESS } from "./products.types";
+
+const initialState: ProductsState = [];
+
+const products = (state = initialState, action: AppAction): ProductsState => {
+  switch (action.type) {
+    case PRODUCTS_FETCH_SUCCESS: {
+      return action.payload;
+    }
+    default:
+      return state;
+  }
+};
+
+export default products;
```

Comment:
I appreciate this file is responsible for changing the state associated with products. But this is perhaps the most confusing step to me at the moment.

Reply:
I'll sort of postpone probing too deeply into this for now then.